lab_1/main.py

```diff
@@ -49,7 +83,28 @@ def get_top_n_words(freq_dict: dict, top_n: int) -> list:
     top_n = 1
     --> ['happy']
     """
-    pass
+    diction = {}
+    values_keys = list(freq_dict.items())
+    for key_value in values_keys:
+        if key_value[1] in diction.keys():
+            diction[key_value[1]].append(key_value[0])
+        else:
+            diction[key_value[1]] = [key_value[0]]
+
+    frequent = list(diction.keys())
+    frequent.sort(reverse = True)
+    collection = []
+    for number in frequent:
+        collection.append(diction[number])
+
+    finished = []
+    for element in collection:
+        if len(element) == 1:
```

Comment:
What is this check for?

Reply:
It is used to make one list which consists of strings.